Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigationBadge.swift

```diff
@@ -0,0 +1,47 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+class BPKHorizontalNavigationBadge: BPKBadge {
+    
+    public var color: UIColor {
+        didSet { updateStyle(with: color) }
+    }
+    
+    init(color: UIColor) {
+        self.color = color
+        super.init(frame: .zero)
+        type = .outline
+        updateStyle(with: color)
+    }
+    
+    required init?(coder: NSCoder) {
```

Comment:
Can we add `@available(*, unavailable)` above this function? This way no one is able to call it by accident.
It does seem however that the BPKBadge class did implement this, so we might want to call `super`?

Reply:
only issue with that is that a color must then be assumed while initializing.
having the color to be optional doesn't really make sense conceptually just to avoid this.
I'll set an accent principal color as a default